### PR TITLE
Cuda 10

### PR DESCRIPTION
**Pull request by kyleroot**
The latest version of Gentle builds against a version of Kaldi doesn't support building against a modern version of CUDA that supports the T4 card we are using.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +37,7 @@ RUN export MAKEFLAGS=' -j8' &&  cd /gentle/ext && \
 
 ADD . /gentle
 RUN cd /gentle && ./install_models.sh
-RUN cd /gentle && python3 setup.py develop
+RUN cd /gentle && pip3 install redis twisted && python3 setup.py develop
```
Comment (jhorman) [inline Dockerfile]: The setup file already has

```
install_requires=['twisted', 'redis', 'python-json-logger'],
```

do we need this here too?

File: ext/k3.cc [hunk 2]
```diff
@@ -96,7 +96,7 @@ int main(int argc, char *argv[]) {
 #ifdef HAVE_CUDA
     fprintf(stdout, "Cuda enabled\n");
     CuDevice &cu_device = CuDevice::Instantiate();
-    cu_device.SetVerbose(true);
+    //cu_device.SetVerbose(true);
```
Comment (jhorman) [inline ext/k3.cc:99]: Same line?

Comment (kyleroot) [reply ext/k3.cc:99]: This was in the instructions in that blog post. I tried it without commenting it out and the build breaks. You can't use `cu_device.SetVerbose(true);`.

File: Dockerfile (hunk 1 above)
Comment (kyleroot) [reply Dockerfile]: No, I removed it. That was a workaround to the broken "pip search" needed implicitly by setuptools on Python 3.5. I just moved everything onto Python 3.9.

**Review by jhorman — APPROVED** [review]

**Pull request merged**